src/app/shared/search/advanced-search/advanced-search.component.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
@GauravD2t 

could you add tests to this class?

Reply:
import css